src/components/button-toggle/button-toggle.style.js

```diff
@@ -0,0 +1,141 @@
+import PropTypes from 'prop-types';
+import styled, { css } from 'styled-components';
+import baseTheme from '../../style/themes/base';
+import { THEMES } from '../../style/themes';
+import StyledButtonToggleClassicLabel from './button-toggle-classic.style';
+
+const StyledButtonToggleContentWrapper = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  height: 100%;
+`;
+
+const StyledButtonToggleLabel = styled.label`
+  display: inline-block;
+  height: 38px;
+  padding: 0 24px;
+  border: 1px solid ${({ theme }) => theme.colors.border};
```

Comment:
Could you wrap the styles in an arrow function to pull in `theme` at the top level? Not too precious about it normally, but there's a lot of repetition going on here